workshops/migrations/0105_migrate_host_to_org.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.6 on 2016-06-28 07:55
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('workshops', '0104_auto_20160629_1247'),
+    ]
+
+    operations = [
+        migrations.RenameModel(
```

Comment:
1. Yes, `workshops_organization` exist in the DB, `workshops.host` doesn't.
2. New relations are created.

Before migration:
```sql
CREATE TABLE "workshops_event" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "start" date NULL, "end" date NULL, "slug" varchar(100) NULL UNIQUE, "url" varchar(100) NULL UNIQUE, "reg_key" varchar(20) NULL, "attendance" integer unsigned NULL, "admin_fee" decimal NULL, "notes" text NOT NULL, "administrator_id" integer NULL REFERENCES "workshops_host" ("id"), "host_id" integer NOT NULL REFERENCES "workshops_host" ("id"), "address" varchar(255) NOT NULL, "country" varchar(2) NULL, "latitude" real NULL, "longitude" real NULL, "venue" varchar(255) NOT NULL, "contact" varchar(255) NOT NULL, "invoice_status" varchar(40) NOT NULL, "completed" bool NOT NULL, "assigned_to_id" integer NULL REFERENCES "workshops_person" ("id"), "instructors_post" varchar(200) NOT NULL, "instructors_pre" varchar(200) NOT NULL, "learners_longterm" varchar(200) NOT NULL, "learners_post" varchar(200) NOT NULL, "learners_pre" varchar(200) NOT NULL, "request_id" integer NULL REFERENCES "workshops_eventrequest" ("id"), "repository_last_commit_hash" varchar(40) NOT NULL, "repository_tags" text NOT NULL, "tag_changes_detected" text NOT NULL, "tags_changed" bool NOT NULL);
```
After migration:
```sql
CREATE TABLE "workshops_event" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "start" date NULL, "end" date NULL, "slug" varchar(100) NOT NULL UNIQUE, "url" varchar(100) NULL UNIQUE, "reg_key" varchar(20) NOT NULL, "attendance" integer unsigned NULL, "admin_fee" decimal NULL, "notes" text NOT NULL, "administrator_id" integer NULL REFERENCES "workshops_host" ("id"), "address" varchar(255) NOT NULL, "country" varchar(2) NULL, "latitude" real NULL, "longitude" real NULL, "venue" varchar(255) NOT NULL, "contact" varchar(255) NOT NULL, "invoice_status" varchar(40) NOT NULL, "completed" bool NOT NULL, "assigned_to_id" integer NULL REFERENCES "workshops_person" ("id"), "instructors_post" varchar(200) NOT NULL, "instructors_pre" varchar(200) NOT NULL, "learners_longterm" varchar(200) NOT NULL, "learners_post" varchar(200) NOT NULL, "learners_pre" varchar(200) NOT NULL, "request_id" integer NULL REFERENCES "workshops_eventrequest" ("id"), "repository_last_commit_hash" varchar(40) NOT NULL, "repository_metadata" text NOT NULL, "metadata_all_changes" text NOT NULL, "metadata_changed" bool NOT NULL, "language_id" integer NULL REFERENCES "workshops_language" ("id"), "host_id" integer NOT NULL REFERENCES "workshops_organization" ("id"));
```
Here `"host_id" integer NOT NULL REFERENCES "workshops_organization" ("id"));` is the addition. All relations are preserved.